go.mod

```diff
@@ -72,13 +72,13 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dimchansky/utfbom v1.1.1 // indirect
 	github.com/docker/cli v23.0.5+incompatible // indirect
-	github.com/docker/distribution v2.8.2-beta.1+incompatible // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
 	github.com/docker/docker v24.0.1+incompatible // indirect
 	github.com/docker/docker-credential-helpers v0.7.0 // indirect
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.5.0 // indirect
 	github.com/eclipse/paho.mqtt.golang v1.3.5 // indirect
-	github.com/emicklei/go-restful/v3 v3.10.0 // indirect
+	github.com/emicklei/go-restful/v3 v3.10.2 // indirect
```

Comment:
Same as https://github.com/open-horizon/anax/pull/3863, please change line 140 from:
```
github.com/emicklei/go-restful/v3 => github.com/emicklei/go-restful/v3 v3.10.0
```
to
```
github.com/emicklei/go-restful/v3 => github.com/emicklei/go-restful/v3 v3.10.2
```
then re-generate `go.sum`